Droid/View/AddConctactActivity.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Support.V7.App;
+using Android.Widget;
+
+namespace KataContactsCSharp.Droid
+{
+	[Activity(Label = "AddConctactActivity")]
+	public class AddConctactActivity : AppCompatActivity, AddContactPresenter.IView
+	{
+		AddContactPresenter presenter;
+
+		internal static void Open(Context context)
+		{
+			var intent = new Intent(context, typeof(AddConctactActivity));
+			context.StartActivity(intent);
+		}
+
+		protected override void OnCreate(Bundle savedInstanceState)
+		{
+			base.OnCreate(savedInstanceState);
+			SetContentView(Resource.Layout.AddContact);
+			InitializePresenter();
+
+			var addButton = FindViewById<Button>(Resource.Id.doneButton);
+			addButton.Click += async (sender, e) =>
+			{
+				var firstname = FindViewById<EditText>(Resource.Id.firstnameEditText).Text;
```

Comment:
I don't know how hard is to include a component into a Xamarin project but this seems like one of those libraries that are really plug&play (you'll only need to call `Cheeseknife.inject(this)` from your activities/fragments), up to you!